code/Sitecore.NSubstitute.UnitTests/FakeItemTester.cs

```diff
@@ -98,6 +108,15 @@ public void Constructor_WhenCalled_HasZeroChildren()
             item.Children.Count.Should().Be(0);
         }
 
+        [Fact]
+        public void Constructor_WhenCalled_HasNoChildren()
+        {
+            Item item = new FakeItem();
+
+            item.GetChildren().Count.Should().Be(0);

```

Comment:
Might use Fluent Asserts -> collection.Should().BeEmpty();

Reply:
Corrected